app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ProjectsController < ApplicationController
+  def show
+    @project = Project.find(params[:id])
+    @challenge = Challenge.find(@project.challenge_id)
```

Comment:
You don't actually need this @challenge instance variable because you can access challenge information from the @project var you have set up!